auth/realm/base/src/main/java/org/wildfly/security/auth/realm/ElytronMessages.java

```diff
@@ -146,4 +146,8 @@
     @LogMessage(level = Logger.Level.DEBUG)
     @Message(id = 13004, value = "JAAS logout failed for principal %s")
     void debugInfoJaasLogoutFailure(Principal principal, @Cause Throwable cause);
+
+    @LogMessage(level = Logger.Level.WARN)
+    @Message(id = 13005, value = "Realm is not available")
```

Comment:
Maybe we could update the message here to something like:

A realm within the distributed realm is unavailable. This realm will be ignored.